api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -137,6 +137,10 @@ type VirtualMachineImageStatus struct {
 	// synced with the vSphere content library item.
 	// +optional
 	ContentVersion string `json:"contentVersion"`
+
+	// Firmware describe the firmware type used by this VirtualMachineImage. eg: bios, efi.
+	// +optional
+	Firmware string `json:"firmware"`
```

Comment:
If a field is optional then you this should be `json:"firmware,omitempty"`. That's true of the `ContentVersion` field above as well.